File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/share_visitor_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ShareVisitorVO:
    """分享访客响应"""
    created_at: str = None
    updated_at: str = None
    visitor_id: str = None
    visit_time: str = None
    user_agent: str = None
    ip_address: str = None
    device_info: str = None
    visit_type: str = None

File: sdkwork-app-sdk-typescript/src/types/share-visitor-vo.ts
/** 分享访客响应 */
export interface ShareVisitorVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 访客ID */
  visitorId?: string;
  /** 访问时间 */
  visitTime?: string;
  /** 用户代理 */
  userAgent?: string;
  /** IP地址 */
  ipAddress?: string;
  /** 设备信息 */
  deviceInfo?: string;
  /** 访问类型 */
  visitType?: string;
}
